src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -109,6 +111,35 @@ private bool IsSentryMessage(LogEventInfo logEvent)
 
             if (logEvent.Level >= LogLevel.Error && logEvent.Exception != null)
             {
+                // don't report uninformative SQLite exceptions
+                // busy/locked are benign https://forums.sonarr.tv/t/owin-sqlite-error-5-database-is-locked/5423/11
+                // The others will be user configuration problems and silt up Sentry
+                var sqlEx = logEvent.Exception as SQLiteException;
+                if (sqlEx != null && (sqlEx.ResultCode == SQLiteErrorCode.Busy ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.Locked ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.Perm ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.ReadOnly ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.IoErr ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.Corrupt ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.Full ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.CantOpen ||
+                                      sqlEx.ResultCode == SQLiteErrorCode.Auth))
+                {
+                    return false;
+                }
+
+                // Swallow the many, many exceptions flowing through from Jackett
+                if (logEvent.Exception.Message.Contains("Jackett.Common.IndexerException"))
+                {
+                    return false;
+                }
+
+                // UnauthorizedAccessExceptions will just be user configuration issues
+                if (logEvent.Exception is UnauthorizedAccessException)
+                {
+                    return false;
+                }
```

Comment:
Can we err on the conservative side and add a non-UI config option that allows us to override this filtering in case of need to support a user? Look at ConsoleLogLevel for example.

Reply:
Can do. Just to be clear this only stops the errors showing up in sentry, they still appear in all the other logs. Is sentry sometimes useful for support?

Reply:
Yea understand. Sentry has saved me a lot of time tracking something down when user doesn’t know what’s going on. Especially in the case they are not on discord.

Reply:
Ah ok.  Will definitely add the option then.

Reply:
Added